### PR TITLE
Updated Quickstart doc

### PR DESCRIPTION
**Pull request by PJUllrich**
### What was wrong?
**Typos:** There were 2 incredibly important typos in the Quickstart doc. 
**Solc Version:** Solidity uses version numbers like `0.4.11`, but in the docs the version was `4.11`. I found it terribly important to add that leading `0.` for clarification. 

### How was it fixed?
Typos: Fixed them.
Solc Version: Added a leading `0.` to the version number.

### CODE REVIEW

**Comment by PJUllrich** [comment]
Ah man, sorry for opening and closing PRs, but I just realized that the damn 'Added Populus to requirements' commit is still in the history of my Fork. Let me try again.

**Comment by pipermerriam** [comment]
> Ah man, sorry for opening and closing PRs, but I just realized that the damn 'Added Populus to requirements' commit is still in the history of my Fork. Let me try again.

It's ok.  Glad to see the contributions.  Let me know if I can help.
